shared/cfg/guest-os/Linux/JeOS/22.x86_64.cfg

```diff
@@ -0,0 +1,25 @@
+- 22.x86_64:
```

Comment:
Honestly I don't think we need to release JeOS for each existing fedora. We had `JeOS.19`, now `JeOS.21` so I'd skip the `JeOS.22` as nobody needed it yet and now he can use `JeOS.23`...